pkg/virt-handler/vm.go

```diff
@@ -1254,6 +1306,7 @@ func (d *VirtualMachineController) defaultExecute(key string,
 		// requiring the phase of the domain and VirtualMachineInstance to be in sync is an
 		// optimization that prevents unnecessary re-processing VMIs during the start flow.
 		phase, err := d.calculateVmPhaseForStatusReason(domain, vmi)
+		log.Log.V(1).Infof("Calculated phase %s", phase)
```

Comment:
do we still need this you think? or was this for debugging?

Reply:
That was debugging, I believe I already removed that.